src/pages/Reservation/reserveForm.jsx

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useNavigate } from "react-router-dom";
+import { fetchHouses } from "../../redux/house/houseSlice";
+import { addReservation } from "../../redux/reservation/reservationSlice";
+
+const ReserveForm = () => {
+  const houses = useSelector((state) => state.houses.houses);
+  const [reservation, setReservation] = useState({
+    start_date: "",
+    end_date: "",
+    house_id: "",
+  });
```

Comment:
- [ ] If I go on to the reserve form, set up the dates and press submit without changing the house:

![image](https://user-images.githubusercontent.com/57421823/233375109-1faa8500-1322-4a48-b608-ae0841868ac6.png)

This will happen:

![image](https://user-images.githubusercontent.com/57421823/233375224-2baabec5-10de-4680-a6ac-58c34450eff6.png)

![image](https://user-images.githubusercontent.com/57421823/233375275-bdf55f57-d92d-498a-9b34-45ea0aa65ee7.png)

![image](https://user-images.githubusercontent.com/57421823/233375318-d00204f2-885f-4d9e-b687-dea731f81f39.png)

## Explanation

The house won't be reserved because a house Id wasn't selected.  

## Solution and tips

This is the default state of the form:

![image](https://user-images.githubusercontent.com/57421823/233375715-82dc3c61-5778-4624-be50-77f7d41b94ce.png)

As you can see, the "Avalon Dublin Station" is already selected, but on submit, if the house is not changed, react will act as if nothing is selected.

Looking at the code:

![image](https://user-images.githubusercontent.com/57421823/233376115-789850e9-3492-451f-ac74-ff30a388da1f.png)

This local state defaults "house_id" as empty.

Just give a default to it with `house_id: "1"` on that local state so the default house that is displayed on the screen is assigned that id.  Of course, this is just the default, so if your users change the house, the id will be changed accordingly:

```javascript
  const [reservation, setReservation] = useState({
    start_date: "",
    end_date: "",
    house_id: "1",
  });
```

Now that you know the cause and the solution, kindly fix this but so your app works as intended, and your UX is improved for your users.

With a default id value, the default house will work (tested it). 💪

Reply:
Change has been made